### ISSUE TITLE
[Modal] not centered and cut at the bottom

### ISSUE THREAD

**Issue by rbluethl** [Semantic-Org/Semantic-UI#5504]
At certain screen resolutions/browser zoom levels it happens that my modal dialogs are not centered and cut at the bottom with a black bar appearing. In that case it's not possible to scroll the dialog, preventing the user from seeing the entire content of the modal. (Very bad, since we are dealing with a payment dialog here.) Has anyone experienced similar issues? I would be glad if you could look into it ASAP. This is kind of blocking on our side. 

Thanks and best regards,
Ronald

Attached you can find a screenshot:
![modal](https://user-images.githubusercontent.com/4116236/27833395-f55a68f8-60d2-11e7-9972-9a92786c6051.png)

**Comment by Banandrew** [comment]
Hi @rbluethl, what are the steps to reproduce it?

**Comment by rbluethl** [comment]
Hi @Banandrew.

I have absolutely no idea, to be honest.
When I locally developed this feature, I had no issues at first.
It first appeared when customers with lower screen resolutions OR higher zoom levels had issues and sent us a screenshot.

It should appear when you play around with different screen resolutions or zoom levels in the browser. However, I can't tell you exactly when it occurs. Sorry for that 👎 

Thanks for looking into it.

Ronald

**Comment by Banandrew** [comment]
@rbluethl I couldn’t reproduce it; unfortunately, we won’t be able to fix it without a solid test case. Would it be possible for you to contact those customers again and ask them about their browser versions, screen resolutions and zoom levels, so that we could investigate further? If they experienced it on mobile, information on their devices and OS versions would also help.

**Comment by rbluethl** [comment]
Thanks for looking into it. Unfortunately I wasn't able to reproduce it myself right away, it first occured at a customer's site. We got rid of the modal for now, so for us it isn't any longer an issue. Sorry that I couldn't describe how to reproduce it in more detail. I played around with different screen resolutions and then sometimes it appeared. Again, sorry and thanks for the quick response. Ronald
